tests/std/tests/VSO_0180466_algorithm_overhauls/test.cpp

```diff
@@ -343,10 +343,25 @@ namespace test_lexicographical_compare {
 
 namespace test_std_copy {
     void test() {
-        array<int, 4> target{{42, 43, 44, 45}};
-        array<int, 2> input{{1729, 1730}};
-        copy(input.begin(), input.end(), target.begin() + 1);
-        assert((target == array<int, 4>{{42, 1729, 1730, 45}}));
+        {
+            array<int, 4> target{{42, 43, 44, 45}};
+            array<int, 2> input{{1729, 1730}};
+            copy(input.begin(), input.end(), target.begin() + 1);
+            assert((target == array<int, 4>{{42, 1729, 1730, 45}}));

```

Comment:
(Pre-existing) Do we really need that many parentheses?

Reply:
For a function-like macro, commas not between parentheses delimit arguments. So absent that outer pair of parens, this `assert` call would have 5 arguments instead of 1. Someone really needs to convince WG14 to make assert variadic so C++ programs don't have to deal with this.

Reply:
I see, what about  extra {} ‽

Reply:
Those are technically "normal" `{}`s. `std::array` is an aggregate that contains an aggregate, and two sets of `{{}}` are necessary to initialize the contained aggregate. Brace elision generally lets us get away with dropping one set, and would probably work here. (Clang was traditionally a huge pain warning about brace elision, I think they've tweaked the diagnostic to allow single-brace `std::array` initialization past.)